index.html

```diff
@@ -75,24 +75,146 @@ <h2>QUESTION:</h2>
           adipisicing elit.
         </p>
         <ul role="list" class="flex flex-wrap gap-5 justify-evenly mt-20">
-          <li class="p__tag m-auto">HTML5</li>
-          <li class="p__tag m-auto">CSS3</li>
-          <li class="p__tag m-auto">Git</li>
-          <li class="p__tag m-auto">Pro</li>
+          <li class="card__tag m-auto">HTML5</li>
+          <li class="card__tag m-auto">CSS3</li>
+          <li class="card__tag m-auto">Git</li>
+          <li class="card__tag m-auto">Pro</li>
         </ul>
       </article>
     </main>
+    <main
+      data-js="bookmarks"
+      class="js-siteElements app-grid__scrollarea hidden"
+    >
+      <article class="card position-relative card__bg mb-20">
+        <button data-js="btnBookmark" class="position-absolute card__bookmark">
+          <svg
+            data-js="imgBookmark"
+            xmlns="http://www.w3.org/2000/svg"
+            width="30"
+            height="30"
+            fill="var(--color-card)"
+            stroke="var(--color-dark)"
+            viewBox="0 0 16 16"
+          >
+            <path
+              d="M2 2v13.5a.5.5 0 0 0 .74.439L8 13.069l5.26 2.87A.5.5 0 0 0 14 15.5V2a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2z"
```

Comment:
I`m curious! Can you tell me what this is for?

Reply:
this is just some data from the svg ;)

Reply:
Ahh thank you! i didn´t use the svg tag.